src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -6190,7 +6192,7 @@ struct service_node_proof_serialized
   uint16_t storage_port;
   uint16_t quorumnet_port;
   uint16_t version[3];
-  uint16_t _padding{0};
+  uint16_t storage_lmq_port; // used to be "padding"
```

Comment:
No need to keep the comment.  The padding was only there to ensure that the struct had no internal padding between members on any sane architecture (checked by the static_assert below).

Reply:
updated